puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Are we introducing a lot of new command line options that we may want to remove once they are available via config file?  If so, it may make sense to wait.  If there aren't a lot of new options and/or we want to continue supporting them at the command line anyway, :shipit: